spec/features/add_space_spec.rb

```diff
@@ -1,9 +1,10 @@
+# frozen_string_literal: true
+
 feature 'add space' do
   scenario 'owners can add spaces' do
-
-    DB.run("TRUNCATE customers RESTART IDENTITY CASCADE")
+    DB.run('TRUNCATE customers RESTART IDENTITY CASCADE')
```

Comment:
Truncating customers should not need to be ran here, it should be called through the Rakefile on every test.